src/concrete/ml/pandas/dataframe.py

```diff
@@ -318,9 +320,16 @@ def save(self, path: Union[Path, str]):
         """
         path = Path(path)
 
-        encrypted_df_dict = self._to_dict()
-        with path.open("w", encoding="utf-8") as file:
-            json.dump(encrypted_df_dict, file)
+        if path.suffix != ".zip":
+            path = path.with_suffix(".zip")
+
+        encrypted_df_dict, evaluation_keys = self._to_dict_and_eval_keys()
+
+        encrypted_df_json_bytes = json.dumps(encrypted_df_dict).encode(encoding="utf-8")
+
+        with zipfile.ZipFile(path, "w") as zip_file:
```

Comment:
can you make sure there is no compression. it's useless to compress the eval key which looks random and it just slows down the saving process

Reply:
yes